subjects/devops/better-cat/README.md

```diff
@@ -0,0 +1,42 @@
+## better-cat
+
+### Instructions
+
+Create a script `better-cat.sh` that will behave similarly to the `cat` command, but with additional functionality.
+
+The script should accept any number of filenames as arguments, and it should print the content of each file to the standard output. If no arguments are provided, the script should print the content of all files in the current directory.
+
+The script should also support the following options:
+
+- `-c`: If this option is provided, the script should exclude all lines that begin with the # character.
+- `-l`: If this option is provided, the script should print the length of the line before each line of output.
+- `-r`: If this option is provided, the script should print a summary line at the end of the output, indicating the total number of lines in the file and the count of characters in the lines printed.
+
+### Usage
+
+```console
+$ echo "This is an example line of text." > example.txt
+$ bash better-cat.sh example.txt
+1: This is an example line of text.
+$ bash better-cat.sh file1.txt file2.txt
+Here would be the content of file1
```

Comment:
I see what you mean here, but it might be confusing since it looks like `Here would be the content of file..` could be the actual content and it is not formatted as the default behaviour - I don't really have a clearer option to suggest but I wanted to share this thought

Reply:
Thank you for sharing your thoughts on this. I understand your concern. However, as this exercise is one of the optional advanced ones, I think it's OK to intentionally leave some room for thought. 

While I agree that it could be more explicitly described, I believe that not giving everything over-specified encourages critical thinking and problem-solving skills. Since there isn't a a clear suggestion, I will leave it as is. Thank you again for your input, and please feel free to reply to this comment and to share any other thoughts or suggestions you may have as I find them very valuable :slightly_smiling_face: